Automattic/Gravatar-SDK-iOS
# Add some avatar customization options in ProfileViewConfiguration

Closes #245 

### Description

I added following properties to `AvatarConfiguration`:

- cornerRadiusCalculator: A closure that calculates the corner radius of avatar based on its length.
- borderWidth
- borderColor
- avatarLength

I also removed some logic from DefaultAvatarProvider that waits until the avatar download ends to set the avatar border. I added that logic but I think it wasn't much necessary. I adjusted the placeholder implementation accordingly to hide the border during placeholder state. 

### Testing Steps

I added snapshot test but I also added a "Customize Avatar" switch to Demo App > Profile View controller. 
 Demo App> Profile Cards can also be checked to see if there are regressions.
